bin/configs/springboot-petstore-server-spring-pageable-delegatePattern-without-j8.yaml

```diff
@@ -6,6 +6,6 @@ templateDir: modules/openapi-generator/src/main/resources/JavaSpring
 delegatePattern: true
 java8: false
 additionalProperties:
-  documentationProvider: springfox
+  documentationProvider: springdoc
```

Comment:
assuming that `java8` should be removed soon this seems to be similar to `bin/configs/spring-boot-delegate.yaml`. Although I'm not sure that in current one `delegatePattern` & `java8` are handled correctly